components/homme/cmake/machineFiles/crusher-gpumpi.cmake

```diff
@@ -0,0 +1,64 @@
+#interactive job
+#bsub -W 2:00 -nnodes 1 -P cli115 -Is /bin/bash
+
+
+#cmake -C ~/acme-fork-lb/components/homme/cmake/machineFiles/summit.cmake -DHOMMEXX_MPI_ON_DEVICE=FALSE ~/acme-fork-lb/components/homme/
+
+#SET (HOMMEXX_MPI_ON_DEVICE FALSE CACHE BOOL "")
+SET (HOMMEXX_CUDA_MAX_WARP_PER_TEAM "16" CACHE STRING  "")
+
+SET (NETCDF_DIR $ENV{OLCF_NETCDF_FORTRAN_ROOT} CACHE FILEPATH "")
+SET (HDF5_DIR $ENV{OLCF_HDF5_ROOT} CACHE FILEPATH "")
+
+SET(BUILD_HOMME_WITHOUT_PIOLIBRARY TRUE CACHE BOOL "")
+
+SET(HOMME_FIND_BLASLAPACK TRUE CACHE BOOL "")
+
+SET(WITH_PNETCDF FALSE CACHE FILEPATH "")
+
+SET(USE_QUEUING FALSE CACHE BOOL "")
+
+SET(ENABLE_CUDA FALSE CACHE BOOL "")
```

Comment:
I do not see variable ENABLE_CUDA used anywhere, only KOKKOS_ENABLE_CUDA is used. I will remove it from all machine files.